contents/stats.md

```diff
@@ -0,0 +1,430 @@
+Since we have only three quantiles, we have Q1, Q2 and Q3 correspoding to the first, second and third quantile, respectively.
+The Q2 (the 0.5 percentile) is also the median and the Q1 and Q3 are the 0.25 and 0.75 percentile.
+The quartiles are important because we often use it to denote a measure of dispersion.
+This measure is known as **interquartile range** (IQR) and is the **difference between the third and first quartile**:
+
+$$ \operatorname{IQR}(x) = \operatorname{Q3}(x) - \operatorname{Q1}(x), $$ {#eq:iqr}
+
+Like the mean absolute deviation, **IQR**, since uses the median and percentiles, is also **robust to outliers**.
+
+As usual, in @fig:plot_dispersion_iqr, we have two data distributions:
+
+- **Upper row**: **normal** distributed data
+- **Lower row**: **non-normal** distributed data
+
+```jl
+fig = plot_dispersion_iqr()
+caption = "Normal and Non-Normal Distributed Data -- Differences Between IQR Measures."
+label = "plot_dispersion_iqr"
+Options(fig; filename=label, caption, label)
+```
+
+Here we can see that the median is *not* influenced by the few influential observations and that the IQR measured by the 1st and 3rd quantiles represents the 50% most probable values of our observations.
+
+### Advice on Dispersion Measures {#sec:stats_dispersion_advice}
+
+You might be wondering: "which dispersion measure shall I use? Variance? Standard Deviation? Mean Absolute Deviation? IQR?".
+Like before, we provide the following advice:
+
+- _Don't_ use **variance**, since it is not an intuitive measure
+- For data that do *not* have outliers, use the **standard deviation**
+- For data that *do* have outliers, use either the **mean absolute deviation** or **IQR**
+- For categorical/nominal data, use some sort of **frequency counter**
+ 
+## Dependence Measures {#sec:stats_dependence}
+
+Now that we covered univariate measures of central tendency and dispersion, we need to talk about bivariate measures.
+**Univariate measures are measures with respect to a single variable**.
+**Bivariate measures are measures with respect to the _relationship_ between two variables**.
+To quantity such relationships we use **dependence measures, which measure somehow the degree of how one variable value depends (or is related) to another variable value**.
+
+There is a distinction between a dependence *measure* and a dependence *relationship*.
+The first is simply a bivariate measure of how one variable is related to the other.
+On the other hand, a dependence relationship implies a profounder relationship, where not only they are related but information regarding one variable gives us information regarding the other.
+
+There two main measures of dependence and they are practically the same since one is the other constrained to the unit range between -1 and 1.
```

Comment:
```suggestion
There are two main measures of dependence and they are practically the same since one is the other constrained to the unit range between -1 and 1.
```